src/datachain/cli/commands/storage/local.py

```diff
@@ -0,0 +1,111 @@
+from pathlib import Path
+
+from datachain.cli.commands.storage.base import CredentialBasedFileHandler
+from datachain.error import DataChainError
+from datachain.lib.dc.storage import read_storage
+
+
+class LocalCredentialsBasedFileHandler(CredentialBasedFileHandler):
+    def cp(self):
+        from datachain.client.fsspec import Client
```

Comment:
this is still way too low level  ... what do we need from to_storage to make it look as simple as:


`read_storage(source_path, update=update).to_storage(....)`

?

Reply:
A few things:
1. read_storage(source_path, update=update).to_storage(....) doesn't work effectively with a single file. Copying test.py to test2.py makes a directory named test.py and create test2.py. 
2. It doesn't return the files it copied from to the files it copied to. We need that to save the upload logs. 


Changing them are out of the scope for this PRs.